apps/nextjs-website/react-components/types/RowText/RowText.types.ts

```diff
@@ -0,0 +1,7 @@
+export interface RowTextProps {
+    title: string; 
+    subtitle?: string; 
+    body?: string; 
+    layout?: 'left' | 'center' | 'right'; 
+    theme?: 'light' | 'dark';
```

Comment:
```suggestion
    layout: 'left' | 'center'; 
    sectionID: string | null;
```
Remove 'right' layout.
Make layout required.
Remove unused theme.
Add sectionID.